### DISCUSSION TITLE
Change the label of the table column header in the list view.

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17164] [Feature Requests]
At the moment I can’t change the label of the table column header in the list view. But that would be nice.

**Comment by smokeyfro** [comment]
Especially when you're wanting to display a subfield from a M2M relational field. I have a plants collection that lists the companions and those to avoid. Both are referencing the same parent plants collection and I'd like to display the list of friends/foes as columns in the list view. 

However when doing so, the column title is set to the field name I've selected to display (in this case Title). Displaying both the friends and the foes in the list results in the column names being set to Title & Title. Not exactly clear which is which. 

I've attached a screenie for further clarification.

![Screenshot from 2023-01-18 17-57-47](https://user-images.githubusercontent.com/109826/213223968-0746bd2c-dfb0-4153-b228-89e47cd64b05.png)

**Comment by rasco** [comment]
Agree!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
